hazelcast/src/main/java/com/hazelcast/client/impl/protocol/codec/custom/SqlErrorCodec.java

```diff
@@ -62,9 +63,15 @@ public static com.hazelcast.sql.impl.client.SqlError decode(ClientMessage.Forwar
             suggestion = CodecUtil.decodeNullable(iterator, StringCodec::decode);
             isSuggestionExists = true;
         }
+        boolean isCauseStackTraceExists = false;
+        java.lang.String causeStackTrace = null;
+        if (!iterator.peekNext().isEndFrame()) {
```

Comment:
@kwart is it ok to return full stacktrace to clients from security perspective? Stacktrace may contain some internals (eg. EE class names) and messages of inner exceptions could contain some more sensitive data (I do not expect passwords, but for example paths, network addresses etc). Do we do that in case of other exceptions? 

IMO returning stacktrace should be explicitly enabled in config and disabled by default (that is standard approach e.g. for web servers) and in Viridian.

Reply:
Currently, we return stack-traces in the exception message type (see `ErrorCodec`), but you are right we should improve the behavior and only send stacktraces from members when it's explicitly enabled. I'll put it into a JIRA within the security roadmap.

Reply:
So my advice is: let's allow it for now, but then fix it together with other error message types - https://hazelcast.atlassian.net/browse/HZ-3250